lldb/source/Plugins/Language/Swift/SwiftHashedContainer.cpp

```diff
@@ -459,20 +459,12 @@ NativeHashedStorageHandler::NativeHashedStorageHandler(
       std::vector<TypeSystemSwift::TupleElement> tuple_elements{
           {g_key, key_type}, {g_value, value_type}};
       m_element_type = type_system->CreateTupleType(tuple_elements);
-      auto *swift_type = 
-          m_element_type.GetCanonicalType().GetOpaqueQualType();
-      auto element_stride = m_element_type.GetByteStride(m_process);
-      if (element_stride) {
+      Status error;
+      if (auto result = runtime->GetMemberVariableOffset(
+              m_element_type, nativeStorage_sp.get(), "1", &error))
```

Comment:
While you're cleaning up, `error` isn't consulted after the call, that argument can be `nullptr`.